flang/test/Driver/integer-overflow.f90

```diff
@@ -0,0 +1,10 @@
+! Test for correct forwarding of integer overflow flags from the compiler driver
+! to the frontend driver
+
+! RUN: %flang -### -fno-strict-overflow %s 2>&1 | FileCheck %s --check-prefixes CHECK,INDUCED
```

Comment:
Do we need both the `CHECK` and `INDUCED` here?

Reply:
As you mentioned, the check for `CHECK` is redundant. I removed it.